lvsfunc/comparison.py

```diff
@@ -169,44 +170,44 @@ class Tile(Comparer):
 
     The arrangement of the clips can be specified with the `arrangement` parameter.
     Rows are specified as lists of ints inside of a larger list specifying the order of the rows.
-    Think of this as a 2-dimensional array of `0` or `1` with `0` representing an empty slot and `1` representing the
+    Think of this as a 2-dimensional array of 0s and 1s with `0` representing an empty slot and `1` representing the
     next clip in the sequence.
 
     If `arrangement` is not specified, the function will attempt to fill a square with dimensions `n x n`
-    where `n` is equivalent to ``math.ceil(math.sqrt(len(clips))``.::
+    where `n` is equivalent to ``math.ceil(math.sqrt(len(clips))``. The bottom rows will be dropped if empty. ::
```

Comment:
I'm just noticing this, but the `::` at the end are meant to be use as a replacement of a single `:` followed by a code block in the next paragraph., e.g. `Examples::\n\n    code here`.

What you would want here is the [code-block directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block)

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

It says if it's preceded by whitespace it's removed, which it is in the docs I built when I was testing this. Not sure how to use the code-block directive in the code, that seems like something that would go into the .rst file, no?

Reply:
> If it is preceded by whitespace, the marker is removed.

Oh, I didn't know about that. If it works, then that's fine, but you could still try the directive (with code highlighting). Iirc, reST is interpreted literally in docstrings, but I could be wrong as it's been a while for me.

Reply:
The way it works currently seems to asssume python for the code block syntax highlighting, which works in this case since the list format would be how you would specify in python.